src/plugins/skills/crafting/tanning-hides/tanning-hides-constants.ts

```diff
@@ -0,0 +1,133 @@
+import { Item } from '@engine/world/items/item';
+import { widgets } from '@engine/config';
+import { itemIds } from '@engine/world/config/item-ids';
+
+interface TanableHide {
+    hideId: number;
```

Comment:
Any reason you're using the numeric ID instead of the string? Know that it's a little more work with containers rn, but it'd be more future-proof to use the strings still.

Reply:
Nope, just an oversight.